src/components/DrsObject.js

```diff
@@ -0,0 +1,24 @@
+import '@fontsource/roboto';
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+    useParams
+  } from "react-router-dom";
+
+function DrsObject(props) {
```

Comment:
let's rename this file. We have the DRS **index** page, this page is showing us everything about a single `DRSObject`. In REST API terminology, this is the `Show` page. So in keeping in convention I would rename to `DrsShow` and place under:

- `src/components/pages/drs/DrsShow.js`